package.json

```diff
@@ -21,6 +21,7 @@
     "react-native-reanimated": "^2.1.0",
     "react-native-safe-area-context": "^3.2.0",
     "react-native-screens": "^3.1.0",
+    "react-navigation": "^4.4.4",
```

Comment:
Could you explain why you use react-navigation version 4 instead of latest (5)?

Reply:
I didn't run any commands to specifically install version 4 here. I suppose it might be due to nvm version. Do you have any idea?

Reply:
I found react-navigation has been added here https://github.com/thongquach-eh/UserRedux/pull/1/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R16. Please remove this line `"react-navigation": "^4.4.4",
 `

Reply:
Thank you for your feedback. I've already fixed ESLint errors remove react navigation 4.4.4